transforms/React-PropTypes-to-prop-types.js

```diff
@@ -135,6 +139,15 @@ module.exports = function(file, api, options) {
           }
         );
 
+        // Add back any nested destructured children.
+        if (nestedPropTypesChildren) {
+          // TODO: This shouldn't just use a template string but the `ast-types` docs were too opaque for me to follow.
```

Comment:
This is the spot that could use some updating from someone with `jscodeshift` experience.